upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.16-v1.10.yaml.template

```diff
@@ -188,8 +189,10 @@ data:
   # [0] http://docs.cilium.io/en/stable/policy/language/#dns-based
   # [1] http://docs.cilium.io/en/stable/install/upgrade/#changes-that-may-require-action
   tofqdns-enable-poller: "{{- if .ToFqdnsEnablePoller -}}true{{- else -}}false{{- end -}}"
+  {{- if not (semverCompare ">=1.11 || ~1.10.4 || ~1.9.10" $semver) }}
```

Comment:
We'll create new templates for 1.11 so testing for 1.11 is not necessary. 1.11 will also require many other changes to the manifest.